compiler/erlang_check.erl

```diff
@@ -231,7 +231,6 @@ read_file_type(Fd) ->
 %% return whether there were errors.
 %% @end
 %%------------------------------------------------------------------------------
--spec check_module(string()) -> ok | error.
```

Comment:
Every function has a type spec and Edoc documentation, so let's keep is that way. They help understanding the code. (I found just now that `print_xref_warnings` doesn't have Edoc... I will fix that.)